docs/guide/onboarding-library.md

```diff
@@ -1,9 +1,158 @@
 # Onboarding Library
+As an Ethereum enabled site developer, sending users offsite to install MetaMask presents challenges. Most notably, you must inform the user to return to your site and refresh their browser after the installation. Your site will detect the user's newly installed MetaMask extension only after that refresh. We at MetaMask care deeply about user experience, and we knew that this workflow needed to be improved.
 
-MetaMask now allows sites to register as onboarding the user, so that the user is redirected back to the initiating site after onboarding. This is accomplished through the use of the [metamask-onboarding library](https://github.com/MetaMask/metamask-onboarding).
+MetaMask now provides a [metamask-onboarding library](https://github.com/MetaMask/metamask-onboarding) designed to improve and simplify the onboarding experience. The new library exposes an API to initiate the onboarding process. In the process, it registers your site as the origin of the onboarding request. MetaMask will check for this origin after the user completes the onboarding flow. If it finds an origin, the final confirmation button of the MetaMask onboarding flow will indicate that the user will be redirected back to your site. The wording of this button helps prevent phishing attempts, by highlighting that a redirect is taking place to an untrusted third party.
```

Comment:
I'm not sure what to think of that last sentence. Do our design decisions aimed at preventing phishing matter to the intended audience here (dapp devs)?

It's kinda neat I guess. It would be OK to cut it though I think.